homeassistant/components/daikin/sensor.py

```diff
@@ -179,11 +214,18 @@ def __init__(
         """Initialize the sensor."""
         self.entity_description = description
         self._api = api
-        self._attr_name = f"{api.name} {description.name}"
+        self._attr_entity_registry_enabled_default = description.enabled_default
+        if description.cross_device:
+            self._attr_name = f"Daikin {description.name}"
```

Comment:
Please see our entity naming guidelines for how to set entity name:
https://developers.home-assistant.io/docs/core/entity#entity-naming

Reply:
This isn't completed.

Reply:
I hadn't understood your point, it should be better now